### PR TITLE
Don't remove the fonts on the build

### PR DESCRIPTION
**Pull request by sbrunner**
fix https://github.com/camptocamp/ngeo/issues/640

### CODE REVIEW

**Comment by ger-benjamin** [comment]
+100 (but you'll must rebase this PR on master...)

File: Makefile [hunk 1]
```diff
@@ -450,6 +450,7 @@ node_modules/angular/angular.min.js: .build/node_modules.timestamp
 	mkdir -p $(dir $@)
 	touch $@
 
+.PRECIOUS: node_modules/font-awesome/fonts/fontawesome-webfont.%
 node_modules/font-awesome/fonts/fontawesome-webfont.%: .build/node_modules.timestamp
 	touch --no-create $@
```
Comment (pgiraud) [inline Makefile]: Can you please use the `-c` to make it work on OSX at the same time?

Comment (sbrunner) [inline Makefile]: We will mix completely deferent things in the same pull request?

Comment (pgiraud) [inline Makefile]: No you're right. Sorry.

**Pull request merged**